### PR TITLE
A little touch on syntax.

### PR DESCRIPTION
**Pull request by aliyilmaz**
The syntax has been updated in light of current capabilities and standards.

### CODE REVIEW

**Comment by aliyilmaz** [comment]
The syntax has been updated again in the light of current capabilities and standards.

File: demos/blog/protected/views/layouts/main.php [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
+	<?=CHtml::cssFile('css/ie.css', 'screen, projection');?>	
 	<![endif]-->
 
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/main.css" />
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/form.css" />
+	<?=CHtml::cssFile('css/main.css');?>	
+	<?=CHtml::cssFile('css/form.css');?>	
 
-	<title><?php echo CHtml::encode($this->pageTitle); ?></title>
+	<title><?=Yii::app()->name;?></title>
```
Comment (samdark) [inline demos/blog/protected/views/layouts/main.php:19]: That's not a good idea:

1. We need page title, not application name.
2. Missing `CHtml::encode()` introduced XSS.

File: demos/blog/protected/views/layouts/main.php [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
 </head>
 
 <body>
 
 <div class="container" id="page">
 
 	<div id="header">
-		<div id="logo"><?php echo CHtml::encode(Yii::app()->name); ?></div>
+		<div id="logo"><?=Yii::app()->name;?></div>
```
Comment (samdark) [inline demos/blog/protected/views/layouts/main.php:27]: Missing `CHtml::encode()` introduced XSS.

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by aliyilmaz** [comment]
Correction of my carelessness. The symbol < has been deleted.

File: demos/blog/protected/views/layouts/main.php (hunk 2 above)
Comment (aliyilmaz) [reply demos/blog/protected/views/layouts/main.php:27]: Thanks.

File: demos/blog/protected/views/layouts/main.php (hunk 1 above)
Comment (aliyilmaz) [reply demos/blog/protected/views/layouts/main.php:19]: Got it, I got it wrong, thank you.

**Pull request closed without merging**